index.js

```diff
@@ -0,0 +1,36 @@
+const calculate = calculationSteps => {
+  calculationSteps = calculationSteps.filter(element => element != '')
+  for (let i = 0; i < calculationSteps.length; i++) {
+    if (calculationSteps[i] === null) {
+      calculationSteps[i] === 0
+    }
```

Comment:
This is unnecessary, you could just typecast in your calculation which you should be doing anyway since `'2' + 2` is 22 not 4.

![image](https://user-images.githubusercontent.com/5775716/78159440-936e4e80-7410-11ea-9d43-c62e6e55e9e7.png)